README.md

```diff
@@ -315,6 +359,41 @@ You can access any URL or endpoint that references this address from your host.
 1. Make a change to component `X`, in its respective release (`X-release`).
 1. Run `make X-release compile images run` to build your changes and run them.
 
+#### Bumping a version in a release (or just make a change)
+
+For this example, lets suppose we want to update a release to a later tag.
+First of all checkout the desired commit:
+
+```
+host> cd src/loggregator/ && git checkout v81
+```
+
+Then from inside the vagrant box regenarate the image for this release:
+
+```
+vagrant> cd scf && make loggregator-release compile images kube run
```

Comment:
It looks like you didn't mean to have `kube run` here, since we're doing it in the next step?